packages/bpk-component-calendar/package.json

```diff
@@ -26,6 +26,7 @@
     "react": "^16.0.0"
   },
   "devDependencies": {
+    "bpk-component-aria-live": "^1.0.38",
```

Comment:
Added this dependency, had a lot of issue defaulting to a specific Artifactory and so didn't push the package-lock.json changes. I'd appreciate some specific attention around this change 🙇

Reply:
No problem - I can push the change.